spec/src/main/asciidoc/preface.asciidoc

```diff
@@ -11,6 +17,10 @@ include::license-{license}.asciidoc[]
 
 Jakarta Contexts and Dependency Injection 3.0 (link:https://jakarta.ee/specifications/cdi/3.0/[CDI 3.0]) is an update to Jakarta Contexts and Dependency Injection 2.0 (link:https://jakarta.ee/specifications/cdi/3.0/[CDI 2.0]).
 
+Contexts and Dependency Injection 4.0 introduce a new feature with the CDI Lite specification.
+
+CDI Lite is a subset of CDI that doesn't require a CDI container at runtime and thus is a good candidate to develop cloud native or resources constraint applications.
```

Comment:
```suggestion
CDI Lite is a subset of CDI that uses the bean discovery process in the build time instead of runtime. Thus is an excellent candidate to develop cloud-native or resources constraint applications.
```